### DISCUSSION TITLE
Country code "HR" aka Croatia unable to fetch postage rates

### DISCUSSION THREAD

**Discussion by Braedencraig** [chitchats/chitchats-api-doc#42] [Q&A] [answered]
I cant seem to fetch postage rates for Croatia when creating a shipment, even when setting the postage_type to "unknown", I still get the "unable to fetch postage rates" error. Any idea what might be causing this? I'm testing this on staging.

**Comment by gshaw** [comment, marked as answer]
A couple of suggestions.

1. Make sure the shipment's weight and dimensions are eligible for USPS First-Class Package International Service.  I don't know the specifics off hand but I think it has to be quite light and small.  Try 1 lb and 6x6x6 inches as I'm seeing a recent shipment with those dimensions.  Unfortunately the system isn't great at giving a reason why it isn't returning any postage types.  We are working on improving that but so far you need to know the limits of each service.
2. Try creating in the UI to see if you see rate.  Unlikely to work but worth a try.

**Reply by Braedencraig** [reply]
It was the weight and dimensions!!! Thank you!!!!
